frmts/tiledb/tiledbcommon.cpp

```diff
@@ -133,9 +133,17 @@ int TileDBDataset::Identify(GDALOpenInfo *poOpenInfo)
             return TRUE;
         }
 
+        const bool bIsS3OrGS =
+            STARTS_WITH_CI(poOpenInfo->pszFilename, "/VSIS3/") ||
+            STARTS_WITH_CI(poOpenInfo->pszFilename, "/VSIGS/");
+        // If this is a /vsi virtual file systems, bail out, except if it is S3 or GS.
+        if (!bIsS3OrGS && STARTS_WITH(poOpenInfo->pszFilename, "/vsi"))
```

Comment:
is there a reason for not checking case insensitive here too? (assuming that STARTS_WITH and STARTS_WITH_CI are what I think they are regarding case sensitivity).

Reply:
Well, the checking for VSIS3 and VSIGS should have been case sensitive from the beginning, but for some reason the original contributor did it case insensitive, so I've kept that check as it in case it was depended upon